postgres/Dockerfile

```diff
@@ -1,11 +1,28 @@
-FROM clearlinux:latest
+FROM clearlinux:latest AS builder
 MAINTAINER william.douglas@intel.com
 
 ARG swupd_args
-
+# Move to latest Clear Linux release to ensure
+# that the swupd command line arguments are
+# correct
 RUN swupd update --no-boot-update $swupd_args
-RUN swupd bundle-add postgresql $swupd_args \
-	&& rm -rf /var/lib/swupd/*
+
+# Grab os-release info from the minimal base image so
+# that the new content matches the exact OS version
+COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
+
+# Install additional content in a target directory
+# using the os version from the minimal base
+RUN source /os-release && \
+    mkdir /install_root \
+    && swupd os-install -V ${VERSION_ID} \
+    --path /install_root --statedir /swupd-state \
+    --bundles=postgresql --no-scripts \
+    && rm -rf /install_root/var/lib/swupd/*
```

Comment:
Can you see about running a `swupd clean --all --path=/install_root` here instead?

Reply:
As checked, the purpose of swupd clean is to clean up statedir. When swupd os-install, statedir is set as /swupd-state, instead of var/lib/swupd.  It will not be copied to the target docker image. So, no clean up should be needed, and would prefer to remove "rm -rf /install_root/var/lib/swupd/*". Thanks!